appveyor.yml

```diff
@@ -16,8 +16,8 @@ install:
     - cinst wget
     - IF EXIST c:\php (SET PHP=0) ELSE (mkdir c:\php)
     - cd c:\php
-    - IF %PHP%==1 appveyor DownloadFile https://github.com/symfony/binary-utils/releases/download/v0.1/php-5.3.3-nts-Win32-VC9-x86.zip
-    - IF %PHP%==1 7z x php-5.3.3-nts-Win32-VC9-x86.zip -y >nul
+    - IF %PHP%==1 curl -fsS -o php-5.3.zip https://windows.php.net/downloads/releases/archives/php-5.3.11-nts-Win32-VC9-x86.zip
```

Comment:
please keep the current way of downloading: unless it's an issue, we shouldn't change unrelated things.

Reply:
it is an issue :confused: for some weird reason, windows.php.net responded with an html page when using `appveyor DownloadFile`, the only way i was able to get this to work was using `curl`